docs/runtime/instantiable-module.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Instantiable Modules"
+---
+
+Substrate runtimes are typically created by composing multiple runtime modules. Many such modules are available for re-use in the Substrate Runtime Module Library, while others may be written with a particular runtime in mind. Many such modules are intended to be re-used in a variety of runtimes. For example most blockchains have notion of a currency and may use the SRML's Balances module. But what if a single runtime wants to use the same module twice? This is exactly what instantiable modules are for.
```

Comment:
I am not personally a fan of how this is written. This is part of the docs for the runtime, we should have already explained what a runtime is, the fact that modules can be written etc...

Something much more succinct I think would be better here:

"Substrate allows you to use the same runtime module multiple times in the same Substrate runtime through the instantiable modules feature" or something like that.

Reply:
I also think the first sentence should clearly communicate the value add of instantiable modules. Something like 

"Instantiable Modules enable the use of the same module in different ways within a runtime. Polkadot's governance uses this feature to incorporate multiple councils for screening proposals. Because each council uses the same module, but chooses different configurations, the instancing pattern described below is especially useful."

Reply:
Let's not start dropping the "P" word unless we are writing something specific about Polkadot.

Reply:
I like using the "P" word. It's the best (only) reference implementation that demonstrates how this feature is actually used in practice. It provides context and, without context, the application of this feature is ambiguous and not clear.